client/lib/wporg/index.js

```diff
@@ -24,7 +27,7 @@ function getWporgLocaleCode( ) {
 		wpOrgLocaleCode;
 
 	currentLocaleCode = i18n.getLocaleSlug();
-	wpOrgLocaleCode = config( 'languages' ).find( function( language ) {
+	wpOrgLocaleCode = find( config( 'languages' ), function( language ) {
```

Comment:
Best to get the fix in place first-and-foremost, but worth pointing out that Lodash's find has a convenient shorthand for this technique:

```js
wpOrgLocaleCode = find( config( 'languages' ), { langSlug: currentLocaleCode } );
```